app/forms/prior_authority/steps/case_contact/solicitor_form.rb

```diff
@@ -9,6 +9,7 @@ class SolicitorForm < ::Steps::BaseFormObject
         validates :contact_first_name, presence: true
         validates :contact_last_name, presence: true
         validates :contact_email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+        validates :contact_email, presence: true, format: { with: RegularExpressions::EMAIL_DOMAIN_REGEXP }
```

Comment:
Cool, looks like my fear about duplicate `validates :contact_email` was unfounded. It would be good to add this to the NSM solicitor form too, otherwise we'll have the same problem when they put invalid emails in that too.

Reply:
Yeah I thought it would override but it just checks all of them which is what we want here.

Reply:
Added to the solicitor form too (hopefully the one you meant?)